File: examen-service/src/main/java/com/backend/examen_service/model/TestAnalyseModel.java
package com.backend.examen_service.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter
public class TestAnalyseModel {
    private Long id;
    private int fkIdAnalyse;
    private String nomTest;
    private String sousEpreuve;
    private Double intervalMinDeReference;
    private Double intervalMaxDeReference;
    private String uniteDeReference;
    private String details;
}

File: src/app/models/test-analyse.model.ts

  export interface TestAnalyse {
    id: number; // Optionnel car généré automatiquement
    fkIdAnalyse: number; // Clé étrangère vers AnalyseTable
    nomTest: string; // Nom du test
    sousEpreuve: string; // Sous-épreuve associée au test
    intervalMinDeReference: number; // Intervalle minimum de référence
    intervalMaxDeReference: number; // Intervalle maximum de référence
    uniteDeReference: string; // Unité de référence (mg/dL, %, etc.)
    details: string; // Détails supplémentaires sur le test
  }
  